packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -3138,6 +3138,8 @@ abstract class Element implements BuildContext, TreeDiagnostics {
           'resulting object.\n'
           'The size getter was called for the following element:\n'
           '  $this\n'
+          'The associated render sliver was:\n'
```

Comment:
Only insofar as I made these changes in order to debug problems while doing the rest of it. :-)